modules/mysql/main.tf

```diff
@@ -35,7 +35,7 @@ resource "google_sql_database_instance" "default" {
     activation_policy           = "${var.activation_policy}"
     authorized_gae_applications = ["${var.authorized_gae_applications}"]
     backup_configuration        = ["${var.backup_configuration}"]
-    ip_configuration            = "${local.ip_configurations["${local.ip_configuration_enabled ? "enabled" : "disabled"}"]}"
+    ip_configuration            = ["${local.ip_configurations["${local.ip_configuration_enabled ? "enabled" : "disabled"}"]}"]
```

Comment:
We also need to update `failover_replica.tf` and `read_replica.tf`: https://github.com/GoogleCloudPlatform/terraform-google-sql-db/blob/02a3a5d1514cd082ab1afc3af6f19192cadeb76c/modules/mysql/read_replica.tf#L44 and https://github.com/GoogleCloudPlatform/terraform-google-sql-db/blob/master/modules/mysql/read_replica.tf#L44

Reply:
Updated.